model/publishing/delivery/tasks/SyncDeliveryEnvironments.php

```diff
@@ -73,10 +73,23 @@ protected function updateDelivery(\core_kernel_classes_Resource $env, \core_kern
                 $OriginDeliveryField => $deliveryUri
             ]);
 
+
+            \common_Logger::d('POST /taoDeliveryRdf/RestDelivery/updateDeferred?'.http_build_query(['searchParams' => $searchParams]));
+            \common_Logger::d(
+                http_build_query(
+                    $this->getPropertiesForUpdating($env, $delivery)
+                )
+            );
+
+
             $request = new Request('POST', '/taoDeliveryRdf/RestDelivery/updateDeferred?'.http_build_query(['searchParams' => $searchParams]));
             $request = $request->withBody(
-                \GuzzleHttp\Psr7\stream_for(http_build_query($this->getPropertiesForUpdating($env, $delivery)
-                )));
+                \GuzzleHttp\Psr7\stream_for(
+                    http_build_query(
```

Comment:
Since calls to `http_build_query` is used 4 times across twenty lines - maybe better to assign to a variable?
So it'd be easier to read this part

Reply:
The file is out of the scope. I reverted the changes.